BizHawk.Emulation.DiscSystem/DiscMountJob.cs

```diff
@@ -147,8 +146,7 @@ void RunBizHawk()
 				catch (DiscJobAbortException) { okParse = false; parseJob.FinishLog(); }
 				if (!string.IsNullOrEmpty(parseJob.OUT_Log)) Console.WriteLine(parseJob.OUT_Log);
 				ConcatenateJobLog(parseJob);
-				if (!okParse)
-					goto DONE;
+				if (!okParse) return RunBizHawk_Done();

```

Comment:
I think we always try to break these into 2 lines at least. But usually this:
```
if (prettyPlease)
{
    OneLinerExpression();
}
```

Reply:
The return has to be two lines anyway so it has to use braces.